packages/cli/src/commands/__tests__/execHandler-vitest.test.js

```diff
@@ -0,0 +1,103 @@
+import { vol } from 'memfs'
+import {
+  vi,
+  afterAll,
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  it,
+} from 'vitest'
+
+import { runScriptFunction as mockRunScriptFunction } from '../../lib/exec'
+import { generatePrismaClient as mockGeneratePrismaClient } from '../../lib/generatePrismaClient'
+import { handler } from '../execHandler'
+
+const redwoodProjectPath = '/redwood-app'
+process.env.RWJS_CWD = redwoodProjectPath
+
+vi.mock('@redwoodjs/project-config', () => {
+  const path = require('path')
+  const BASE_PATH = '/redwood-app'
+  return {
+    resolveFile: (path) => path,
+    getPaths: () => {
+      return {
+        base: BASE_PATH,
+        scripts: path.join(BASE_PATH, 'scripts'),
+      }
+    },
+  }
+})
```

Comment:
@Tobbe, I'm just getting back to this PR, and noticed that after the tests were switched from jest to vitest that they're not working. 

I could use some help figuring out the issue, which I think has to do with setting up mocks for `project-config` based on what I'm seeing other tests do, but I haven't been able to get it working.

For reference, I temporarily created two different implementations of the same test case:
execHandler-vitest.test.js
execHandler-jest.test.js

The jest test works, but I'm getting an error with vitest: _"Could not find a "redwood.toml" file, are you sure you're in a Redwood project?"_

Is this something you've seen before, or do you have an ideas on what changes I should make to get the vitest version running?

Reply:
I've tried a few different variations of declaring this mock, such these, but I get the same error.

```javascript
vi.mock('@redwoodjs/project-config', async (importOriginal) => {
  const path = require('path')
  const originalProjectConfig = await importOriginal()
  const BASE_PATH = '/redwood-app'
  return {
    ...originalProjectConfig,
    getPaths: () => {
      return {
        base: BASE_PATH,
        scripts: path.join(BASE_PATH, 'scripts'),
      }
    },
  }
})
```

and

```javascript
vi.mock('@redwoodjs/project-config', () => {
  const BASE_PATH = '/redwood-app'
  return {
    resolveFile: (path) => path,
    getPaths: () => {
      return {
        base: BASE_PATH,
        scripts: path.join(BASE_PATH, 'scripts'),
      }
    },
    getConfigPath: () => {
      return 'redwood.toml'
    },
    getConfig: () => {
      return {}
    },
  }
})
```